recipe/meta.yaml

```diff
@@ -4,40 +4,38 @@
 {% set name = base_name + major_version %}
 
 package:
-  name: {{ base_name }}
+  name: {{ name }}
   version: {{ version }}
 
 source:
   - url: https://github.com/ignitionrobotics/ign-plugin/archive/ignition-plugin_{{ version }}.tar.gz
     sha256: ce089cd6924c460d16591f4d3983b282f2f05ed8b5588f7ede9a91dbd43bd5d7
 
 build:
-  number: 2
-  skip: true  # [win and vc<14 or unix]
+  number: 3
+  skip: true  # [win and vc<14]
   run_exports:
     - {{ pin_subpackage(name, max_pin='x') }}
 
-outputs:
-  - name: {{ name }}
-    version: {{ version }}
-    script: bldpkg.bat
-    requirements:
-      build:
-        - {{ compiler('cxx') }}  # [not win]
-        - {{ compiler('c') }}    # [not win]
-        - vs2017_win-64          # [win64]
-        - cmake
-        - ninja
-        - pkg-config
-      host:
-        - libignition-cmake2
-        - dlfcn-win32  # [win]
-      run:
-        - dlfcn-win32  # [win]
-    test:
-      commands:
-        - test -f ${PREFIX}/include/ignition/plugin{{ major_version }}/ignition/plugin.hh  # [not win]
-        - if exist %PREFIX%\\Library\\include\\ignition\plugin{{ major_version }}\ignition\plugin.hh (exit 0) else (exit 1)  # [win]
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not win]
+    - {{ compiler('c') }}    # [not win]
+    - vs2017_win-64          # [win64]
+    - cmake
+    - ninja
+    - pkg-config
+  host:
+    - libignition-cmake2
+    - dlfcn-win32  # [win]
+  run:
+    - dlfcn-win32  # [win]
```

Comment:
Probably unrelated, but as dlfcn-win32 has the run_exports (https://github.com/conda-forge/dlfcn-win32-feedstock/blob/master/recipe/meta.yaml#L15) probably we can drop this?